src/pages/landing/landing.jsx

```diff
@@ -76,14 +85,14 @@ const Landing = () => {
             </p>
             <Link to="/account/login">
               <button>Get Started</button>
-            </Link>
+            </Link>Features
```

Comment:
``features`` should be removed as I believe it's a mistake

Reply:
Thank you.

Reply:
I have made the following changes:
- remove features link 
- fix navigation links in hamburger menu